modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/Main.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.fs2
+
+import cats.syntax.flatMap._
+
+import cats.effect.{Blocker, Resource, ExitCode, IOApp, IO}
+
+object Main extends IOApp {
+  def run(args: List[String]): IO[ExitCode] =
+    EnrichJobConfig.command.parse(args) match {
+      case Right(cfg) =>
+        val environment = for {
+          blocker <- Blocker.apply[IO]
```

Comment:
Why are we initializing `blocker` here rather than [here](https://github.com/snowplow/enrich/pull/328/files#diff-b020b3f46b164eb407a4c7553f253befR57-R68) ?

Reply:
No reason I think - you're probably right, it look better in `Environment`.